lib/declarative_authorization/controller/grape.rb

```diff
@@ -57,13 +57,18 @@ def logger
             ::Rails.logger
           end
 
-          protected
-
           def api_class
             if options[:for].respond_to?(:base)
-              options[:for].base # Grape >= 1.2.0 controller
+              # Grape >= 1.2.0 endpoint
+              # Authorization::Controller::Grape can be included into either Grape::API
+              # or Grape::API::Instance, so we need to check both.
+              [
```

Comment:
Is the order significant? It seems we used to check `options[:for].base` first when it used to be in `ControllerPermission`.

Reply:
Its only ever going to be one or the other.